go.mod

```diff
@@ -18,7 +18,6 @@ require (
 	github.com/dlclark/regexp2 v1.2.0 // indirect
 	github.com/docker/docker v1.4.2-0.20180625184442-8e610b2b55bf
 	github.com/dop251/goja v0.0.0-20200721192441-a695b0cdd498
-	github.com/dvyukov/go-fuzz v0.0.0-20200318091601-be3528f3a813 // indirect
```

Comment:
Sure we can drop this?

Reply:
Hmmm, It's modified by go mod itself. Let me check

Reply:
Looks like it's dropped. But no compilation failure. I think it's fine?

I also run the `go mod tidy`, it even deletes two more lines...

Reply:
Ok, I guess it'll pop back if needed